src/app/services/startup/startup.service.ts

```diff
@@ -362,6 +362,14 @@ export class ArlasStartupService {
             const useAuthent = !!settings && !!settings.authentication && !!settings.authentication.use_authent;
             if (useAuthent) {
                 const authService: AuthentificationService = this.injector.get('AuthentificationService')[0];
+                const usePersistence = (!!settings && !!settings.persistence && !!settings.persistence.url
+                    && settings.persistence.url !== '' && settings.persistence.url !== NOT_CONFIGURED);
+                if (usePersistence) {
+                    // To open reconnect dialog on silent refresh failed
+                    authService.silentRefreshErrorSubject.subscribe(error => this.persistenceService.list('config.json', 10, 1, 'asc'))
```

Comment:
why do we call the list method after a silent refresh fail ?

Reply:
To deal with anonymous mode which will return a silent refresh faildes but a response 200 to the server.